### PR TITLE
Refactor linelists from QTable -> Table

### PR DESCRIPTION
**Pull request by profxj**
Despite my original fascination with QTable's, I've come
to consider them a greater pain than good.

This PR removes the use of QTable's in linelists in linetools.

tests updated
no doc changes aside from those in the modules

It is possible a Notebook or two will break, but I doubt it.

### CODE REVIEW

File: linetools/lists/parse.py [hunk 1]
```diff
@@ -853,16 +851,16 @@ def update_fval(table, verbose=False):
...
+    if mn > 0.05:  # Ang
         _, new_row = line_data()
         new_row['Z'] = 33
         new_row['ion'] = 2
         new_row['wrest'] = 1355.934
         new_row['name'] = 'AsII 1355'
         new_row['f'].mask = True
         # Stack
-        table = QTable(vstack([Table(table), new_row]))
+        table = Table(vstack([Table(table), new_row]))
```
Comment (ntejos) [inline linetools/lists/parse.py]: leave only `vstack([table, new_row])`

File: linetools/lists/parse.py [hunk 2]
```diff
@@ -932,7 +930,7 @@ def _write_ref_ISM_table():
     euv = LineList('EUV', use_ISM_table=False)
     hi = LineList('HI', use_ISM_table=False)
 
-    # need a Table, not QTable to write
+    # need a Table to write
     tab = Table(ism._data.copy())
```
Comment (ntejos) [inline linetools/lists/parse.py]: I assume this is redundant too; `ism._data` is Table right?

File: linetools/lists/linelist.py [hunk 3]
```diff
@@ -117,9 +118,9 @@ def name(self):
 
     @property
     def wrest(self):
-        """ Return the rest wavelengths
+        """ Return the rest wavelengths as a Quantity array
         """
-        return self._data['wrest']
+        return Quantity(self._data['wrest'])
```
Comment (profxj) [inline linetools/lists/linelist.py]: The Table does have units.  And by using 
Quantity here, they get 'ingested'.  Rather nice, eh?

File: linetools/lists/linelist.py [hunk 4]
```diff
@@ -794,33 +807,33 @@ def available_transitions(self, wvlims, n_max_tuple=None, min_strength=0.):
...
 
-            # need to deal with dict vs QTable format now
+            # need to deal with dict vs Table format now
             if isinstance(aux, dict):
-                aux = self.from_dict_to_qtable(aux)
+                aux = self.from_dict_to_table(aux)
 
-            # convert to Table because QTable does not like vstack
+            # convert to Table because Table does not like vstack
             output = vstack([output, Table(aux)])
```
Comment (profxj) [inline linetools/lists/linelist.py]: agreed.  edited.

File: linetools/lists/linelist.py [hunk 5]
```diff
@@ -794,33 +807,33 @@ def available_transitions(self, wvlims, n_max_tuple=None, min_strength=0.):
...
 
         # Deal with output formatting now
         # if len==1 return dict
         if len(output) == 1:
-            return self.from_qtable_to_dict(output)
+            return self.from_table_to_dict(output)
         else:
-            return QTable(output)
+            return Table(output)
```
Comment (profxj) [inline linetools/lists/linelist.py]: yup.  edited

File: linetools/lists/parse.py (hunk 1 above)
Comment (profxj) [inline linetools/lists/parse.py]: agreed.  done

File: linetools/lists/parse.py (hunk 2 above)
Comment (profxj) [inline linetools/lists/parse.py]: yup

File: linetools/lists/linelist.py (hunk 3 above)
Comment (ntejos) [inline linetools/lists/linelist.py]: Not very intuitive but nice, I agree.

**Comment by profxj** [comment]
Are we good to go, @ntejos ?

If so, please merge.

**Comment by ntejos** [comment]
Well, there is a test failing at the moment... any idea?

**Comment by profxj** [comment]
It was a Travis thing.

Merging.

**Pull request merged**